tools/traffic/config/flags.go

```diff
@@ -0,0 +1,262 @@
+package config
```

Comment:
are these moved from `/tools/traffic/flags.go`? or are these being added in addition to the existing ones?

Reply:
In the short term, the new `flags.go` file will duplicate the original file in the `master` branch, but will not be used for anything. When the rest of the code from [this branch](https://github.com/Layr-Labs/eigenda/pull/666) merges, the new `flags.go` will entirely replace the old one.

I'm merging this way so that I can split off the config changes into a stand alone PR. It can't replace the old flags because it's not fully backwards compatible.